transactions.go

```diff
@@ -361,6 +382,104 @@ func (lw *LibWallet) DecodeTransaction(txHash []byte) (string, error) {
 	return string(result), nil
 }
 
+func (lw *LibWallet) queryTransactions(query storm.Query) (string, error) {
+	var transactions []Transaction
+
+	err := query.Find(&transactions)
+	if err != nil {
+		return "", nil
+	}
+
+	jsonEncodedTransactions, err := json.Marshal(&transactions)
+	if err != nil {
+		return "", err
+	}
+
+	return string(jsonEncodedTransactions), nil
+}
+
+func (lw *LibWallet) countTransactions(query storm.Query) (int, error) {
+
+	count, err := query.Count(&Transaction{})
+	if err != nil {
+		return -1, err
+	}
+
+	return count, nil
+}
+
+func (lw *LibWallet) FilterTransactions(txFilter int32) (string, error) {
+	query := lw.parseTxFilter(txFilter)
+	return lw.queryTransactions(query)
+}
```

Comment:
There don't seem to be any other call to `lw.queryTransactions`, can probably implement the function here directly.